spec/requests/partners_requests_spec.rb

```diff
@@ -124,4 +124,75 @@
       expect(response).to have_http_status(:found)
     end
   end
+
+  describe "PUT #deactivate" do
+    let(:partner) { create(:partner, organization: @organization) }
+
+    context "when the partner successfully deactivates" do
+      it 'performs a PUT request' do
```

Comment:
```suggestion
      it 'changes the partner status to deactivated and redirects with flash' do
```
The `it` should contain information about what is expected. We do know it performs a `PUT` but that isn't what we are testing :)